itou/templates/account/account_inactive.html

```diff
@@ -0,0 +1,15 @@
+{# django-allauth template override. #}
+{% extends "layout/content_small.html" %}
```

Comment:
pour ma curiosité comment marche ce tour de magie? Tu overwrite magiquement juste en te basant sur le filename `account_inactive`?

https://github.com/pennersr/django-allauth/blob/master/allauth/templates/account/account_inactive.html

Reply:
C'est le mécanisme du backend de templates de Django qui permet ça : https://docs.djangoproject.com/en/dev/howto/overriding-templates/#overriding-from-the-project-s-templates-directory

C'est documenté ici dans `allauth` :

- https://django-allauth.readthedocs.io/en/latest/templates.html#overridable-templates
- https://django-allauth.readthedocs.io/en/latest/views.html#login-account-login

Reply:
👍